modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration_{context}"]
+= NotificationsConfiguration custom resource properties
```

Comment:
Just a small comment, but the ID should now correspond with the title :+1:

Reply:
Fixed.